src/modules/flow/selector/selector.json

```diff
@@ -0,0 +1,124 @@
+          "methods": {
+            "process": "select_set"
+          },
+          "name": "SELECT"
+        },
+        {
+          "data_type": "boolean",
+          "description": "Make the interaction possible, otherwise don't produce strings or process NEXT/PREVIOUS/SELECT inputs. In other words, enabled/disable the node at run-time. The node starts in the enabled state. Note that this does not affect the use of the ADD_ITEM/CLEAR ports.",
+          "methods": {
+            "process": "enable_set"
+          },
+          "name": "ENABLE"
+        }
+      ],
+      "methods": {
+        "close": "selector_close",
+        "open": "selector_open"
+      },
+      "name": "selector",
+      "options": {
+        "members": [
+          {
+            "data_type": "int",
+            "description": "The number of available columns to lay out the output string with. It must be a positive integer.",
+            "name": "columns"
+          },
+          {
+            "data_type": "int",
+            "description": "The number of available rows to lay out the output string with. It must be a positive integer.",
+            "name": "rows"
+          },
+          {
+            "data_type": "string",
+            "default": null,
+            "description": "The title string, to be available for referencing when evaluating the format one by means of the '{title}' tag. Any line breaking characters is this string will be translated to a space instead.",
+            "name": "title"
+          },
+          {
+            "data_type": "string",
+            "default": "* ",
+            "description": "If set, selected items will be laid out with this string pre-fixed, separated by a space",
+            "name": "selection_marker"
+          },
+          {
+            "data_type": "string",
+            "default": "> ",
+            "description": "If set, selected items will be laid out with this string pre-fixed, separated by a space",
+            "name": "cursor_marker"
+          },
+          {
+            "data_type": "string",
+            "default": "{value}",
+            "description": "The format string to produce the final output with. The syntax is a free-form string with one '{value}' sub-string in it, to be replaced by the actual string items that came in the ITEMS port.",
+            "name": "format"
+          },
+          {
+            "data_type": "boolean",
+            "default": false,
+            "description": "If true, this makes a circular menu, where after the last option it goes back to the first, in any direction (next or previous).",
+            "name": "circular"
+          }
+        ],
+        "version": 1
+      },
+      "out_ports": [
+        {
+          "data_type": "string",
+          "description": "The formatted, selected entry. This is meant to feed real display nodes (like grove/lcd-string).",
+          "name": "STRING"
+        }
+      ],
```

Comment:
Not sure if I understood node purpose, but shouldn't it output raw value too? To use on the flow?